### PR TITLE
lock mobile portrait mode

### PR DESCRIPTION
**Pull request by 7opf**

### CODE REVIEW

File: identity-enabler/verifier-mobile-app/android/app/src/main/java/com/hello/svelte/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package org.iota.zebra;
```
Comment (jmcanterafonseca-iota) [inline identity-enabler/verifier-mobile-app/android/app/src/main/java/com/hello/svelte/MainActivity.java:1]: be careful this class does no longer exist

**Review by jmcanterafonseca-iota — CHANGES_REQUESTED** [review]

**Comment by jmcanterafonseca-iota** [comment]
this Pull Request has to be rebased

**Comment by jmcanterafonseca-iota** [comment]
@7opf the verifier application is missing

**Comment by 7opf** [comment]
> @7opf the verifier application is missing

@jmcanterafonseca-iota it's already configured: https://github.com/jmcanterafonseca-iota/zebra-iota-edge-sdk/blob/lock_rotation/identity-enabler/verifier-mobile-app/android/app/src/main/java/org/iota/zebra/verifier/MainActivity.java

**Pull request merged**